tools/wasme/cli/pkg/deploy/istio/istio_provider.go

```diff
@@ -450,9 +454,24 @@ func (p *Provider) makeIstioEnvoyFilter(filter *v1.FilterSpec, image pull.Image,
 		return nil, err
 	}
 
+	var patchContext networkingv1alpha3.EnvoyFilter_PatchContext
+	switch strings.ToLower(filter.GetPatchContext()) {
+	case PatchContextAny:
+		patchContext = networkingv1alpha3.EnvoyFilter_ANY
+	case PatchContextInbound, "":
+		// include empty string in this case for backword compatibility
+		patchContext = networkingv1alpha3.EnvoyFilter_SIDECAR_INBOUND
+	case PatchContextOutbound:
+		patchContext = networkingv1alpha3.EnvoyFilter_SIDECAR_OUTBOUND
+	case PatchContextGateway:
+		patchContext = networkingv1alpha3.EnvoyFilter_GATEWAY
+	default:
+		return nil, errors.Errorf("unknown patch context %v, must be one of %v, %v, %v, and %v", filter.GetPatchContext(), PatchContextGateway, PatchContextInbound, PatchContextOutbound, PatchContextGateway)
```

Comment:
Rather than listing them all out here, could we use the array created in `options.SupportedPatchContexts` so that we have one source of truth?

Reply:
I totally agree with you :-) I resolved this problem in the commit f80783042cd7e173e24a262a8d64b5c4325c99f3. Could you check it, please?